Module#02/api/src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+spring.datasource.driverClassName=org.postgresql.Driver
+spring.datasource.url=jdbc:postgresql://oregon-postgres.render.com:5432/reddit_5qs5
+spring.datasource.username=reddit_5qs5_user
+spring.datasource.password=9pnjM54Tkgtiqwdfq20yaSXN4YNNEBuQ
+
+spring.profiles.active=dev
```

Comment:
H2 embedded memory. Because I am as a new developer just want to run your app after cloning from git repo

Reply:
the DataSource has been changed